lib/src/text-input/TextInput.tsx

```diff
@@ -515,6 +518,7 @@ const DxcTextInput = React.forwardRef<RefType, TextInputPropsType>(
                     event.stopPropagation();
                   }}
                   title={action.title ?? action.title}
```

Comment:
Why are we checking nullish/undefined values? This would behave the same as `title={action.title}` because when the prop is not defined, title will also be undefined and that is what we are trying to achieve.

Reply:
Agree. Changed!